wdl2cwl/main.py

```diff
@@ -666,23 +666,23 @@ def convert(workflow: str) -> str:
     if "cpu" in ast.task_runtime:
 
         cpu: Union[str, int] = ""
-        if '"' not in ast.task_runtime["cpu"]:
-            if ast.task_runtime["cpu"] in input_names:
-                cpu = f"$({inputs(ast.task_runtime['cpu'])})"
-            elif ast.task_runtime["cpu"].isnumeric():
-                cpu = int(ast.task_runtime["cpu"])
-            elif "+" in ast.task_runtime["cpu"]:
-                temp = ast.task_runtime["cpu"].split("+")
-                append_str = "$("
-                for i in range(0, len(temp)):
-                    if temp[i] in input_names:
-                        append_str += inputs(temp[i])
-                    elif temp[i].isnumeric():
-                        append_str += temp[i]
-                    if i != len(temp) - 1:
-                        append_str += " + "
-                append_str += ")"
-                cpu = append_str
+        cpu_value = ast.task_runtime["cpu"].strip("'\" ")
```

Comment:
The suggested fix is implemented here :point_up: the rest of the code is pretty much the same.